### PR TITLE
Add kops-aws-1.6 job to the release-1.6 dashboard

### PR DESCRIPTION
**Pull request by ixdy**
x-ref #2029

cc @ethernetdan @enisoc @marun

### CODE REVIEW

**Comment by ixdy** [comment]
cc @justinsb

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1851,14 +1851,16 @@ dashboards:
...
+  - name: soak-gce-1.6-deploy
     test_group_name: ci-kubernetes-soak-gce-1.6-deploy
-  - name: ci-kubernetes-soak-gce-1.6-test
+  - name: soak-gce-1.6-test
     test_group_name: ci-kubernetes-soak-gce-1.6-test
-  - name: ci-kubernetes-soak-gci-gce-1.6-deploy
+  - name: soak-gci-gce-1.6-deploy
     test_group_name: ci-kubernetes-soak-gci-gce-1.6-deploy
-  - name: ci-kubernetes-soak-gci-gce-1.6-test
+  - name: soak-gci-gce-1.6-test
```
Comment (zmerlynn) [inline testgrid/config/config.yaml]: I assume these are intentional "along the way" changes?

Comment (ixdy) [inline testgrid/config/config.yaml]: yeah, this was a mistake when I created them.

**Pull request merged**